app.js

```diff
@@ -9,169 +9,200 @@ function testEvent(){
-    counterSeven = counterSeven + 1;
-    if(userNumber === myNumber){
-      alert('Great job ' + userName +', you guessed my number!');
+    if (answerFour === myAnswerFour) {
+      alert('Very well done!');
       tallyScore = tallyScore + 1;
+    } else {
+      alert('Wrong! On to the next question!');
+    };
 
-      break;
-    }
-    if(counterSeven === 4){
-      alert(userName + ', You used all your guesses, NO SOUP FOR YOU!')
-    }
-    else if(userNumber > myNumber){
-      alert('That number is to high!');
-      userNumber = parseInt(prompt('What number am I thinking of?'));
-    }
-    else if(userNumber < myNumber){
-      alert('That number is too low!');
-      userNumber = parseInt(prompt('What number am I thinking of?'));
-    }
-    else{
-      alert("Please enter a number.");
-      userNumber = parseInt(prompt('What number am I thinking of?'));
-    }
-  };
-
-
-  for(var i = 0; i < 6; i++){
-    var myPlacesAnswers = ['ENGLAND', 'FRANCE', 'CANADA', 'USA', 'UNITED STATES OF AMERICA', 'US', 'UNITED STATES'];
-    var userPlacesAnswers = prompt('Last one! Can you guess what countries I\'ve lived in?');
-    userPlacesAnswers = userPlacesAnswers.toUpperCase();
+    console.log("Question five complete");
+  }
+  questionFive();
+  //
+  function questionSix() {
+    var userResponsePet = prompt('What is one of my pet\'s name?');
+    userResponsePet = userResponsePet.toUpperCase();
+
+    if(answersSixOptions.indexOf(userResponsePet) !== -1){
+      alert('Awesome job!!');
+      tallyScore = tallyScore + 1;
+    } else {
+      alert('You\'re wrong!');
+    };
 
-    if(myPlacesAnswers.indexOf(userPlacesAnswers) !== -1){
-      alert('Great job! your answer was correct!');
+    console.log("Question five complete")
+  }
+  questionSix();
+  //
+  function questionSeven() {
+    alert("Time for a two part answer question! You have to get both answers to be right! What two languages do I speak?");
+
+    var userLanguageResponse1 = prompt('Enter answer one here.');
+    userLanguageResponse1 = userLanguageResponse1.toUpperCase();
+
+    var userlanguagResponse2 = prompt('Enter answer two here.');
+    userlanguagResponse2 = userlanguagResponse2.toUpperCase();
+
+    if (userLanguageResponse1 === 'FRENCH'){
+      alert('Very well done, you got the first one right!');
+    } else if (userLanguageResponse1 === 'ENGLISH') {
+      alert('Very well done, you got the first one right!');
+    } else{
+      alert('Wrong! Let\'s check your other answer!');
+    };
+
+    if (userlanguagResponse2 === 'FRENCH'){
+      alert('Very well done, you got the second one right!');
+    } else if (userlanguagResponse2 === 'ENGLISH') {
+      alert('Very well done, you got the second one right!');
+    } else{
+      alert('Wrong!');
+    };
+
+    if ((userLanguageResponse1 === 'FRENCH') && (userlanguagResponse2 === 'ENGLISH')) {
+      alert('Awesome you got them both right!');
       tallyScore = tallyScore + 1;
```

Comment:
This would actually be an awesome place to increment the tallyScore by 2 if they got both right. :)